issuing_dispute.go

```diff
@@ -44,11 +44,11 @@ type IssuingDisputeEvidenceParams struct {
 
 // IssuingDisputeParams is the set of parameters that can be used when creating or updating an issuing dispute.
 type IssuingDisputeParams struct {
-	Params      `form:"*"`
-	Amount      *int64                        `form:"amount"`
-	Evidence    *IssuingDisputeEvidenceParams `form:"evidence"`
-	Reason      *string                       `form:"reason"`
-	Transaction *string                       `form:"transaction"`
+	Params              `form:"*"`
+	Amount              *int64                        `form:"amount"`
+	Evidence            *IssuingDisputeEvidenceParams `form:"evidence"`
+	Reason              *string                       `form:"reason"`
+	DisputedTransaction *string                       `form:"disputed_transaction"`
```

Comment:
This got changed in pay-sever so I'm just updating it here to pass the test

Reply:
This makes it a breaking change though so tagging as such!

Reply:
Thanks Remi!